src/main/java/de/comroid/util/ChannelUtils.java

```diff
@@ -43,4 +46,13 @@
                     updater.update().exceptionally(ExceptionLogger.get());
                 });
     }
+
+    public static Optional<Message> previous(Message before) {
+        return before.getChannel()
```

Comment:
return before.getMessagesBefore(1);

Reply:
Thanks for taking your time to also comment here with a fix suggestion!

Reply:
It is actually just a pointer, as it returns a completable future, not an optional :-)